docs/overview/overview.rst

```diff
@@ -25,53 +25,53 @@ Contact management
 
 Build a rich and functional database of Contacts for your business, group them together based on your business needs or based on shared attributes.
 
-- :doc:`/contacts/manage_contacts` Individuals who have visited your websites or interacted with your business in some way. They may be known or unknown visitors. Contacts are the primary factor of a marketing automation platform.
+- :doc:`/contacts/manage_contacts` - Known or unknown individuals who have visited your websites or interacted with your business in some way. Contacts are the primary factor of a marketing automation platform.
 
-- :doc:`/segments/manage_segments` A group of your Contacts that share certain attributes. For example, you may segment Contacts based on a contact's location. This may help you easily send out location-based offers or promotions to this specific group of Contacts.
+- :doc:`/segments/manage_segments` - A group of your Contacts that share certain attributes. For example, you may Segment Contacts based on a Contact's location. This may help you easily send out location-based offers or promotions to this specific group of Contacts.
```

Comment:
I think we should use lower case in 'you may segment Contacts' as we are describing the action of adding them into a segment, rather than the thing itself.

Reply:
I get.